addons/pos_hr/static/src/js/models.js

```diff
@@ -64,6 +33,23 @@ class PosHrPosModel extends PosGlobalState {
             this.PRODUCT_SCREEN.numpadMode = 'quantity';
         }
     }
+
+    /**{name: null, id: null, barcode: null, user_id:null, pin:null}
+     * If pos_hr is activated, return {name: string, id: int, barcode: string, pin: string, user_id: int}
+     * @returns {null|*}
+     */
+    get_cashier() {
+        if (this.config.module_pos_hr) {
+            return this.cashier;
+        }
+        return super.get_cashier();
+    }
+    get_cashier_user_id() {
+        if (this.config.module_pos_hr) {
+            return this.cashier.user_id ? this.cashier.user_id : null;
```

Comment:
is `user_id` a tuple (id, display_name) or just the user id?

Reply:
Just id. I set `load=False` in the search params of `hr.employee`

Reply:
I see. Maybe we have to default to `load=False` later. We normally don't need the `display_name` in the Many2one field access.

Reply:
Yeah that wold be good, but I've seen case where we also need the display_name, I think I've seen this for country. Anyways this is still a solvable issue